metnet/layers/DilatedCondConv.py

```diff
@@ -15,6 +18,16 @@ def __init__(
         kernel_size: int = 3,
         activation: nn.Module = nn.ReLU(),
     ):
+        """
+        Initialize the Convolution Block with channels, kernel size and the activation function.
+
+        Args:
+        input_channels: int,
```

Comment:
These should be more descriptions of the arguments, not just a copy of the default args, for example:
```suggestion
        input_channels: Number of inputs channels
```